tests/tests_change_mount.yml

```diff
@@ -0,0 +1,41 @@
+---
+- hosts: localhost
+  become: true
+  vars:
+    mount_location_before: '/opt/test1'
+    mount_location_after: '/opt/test2'
+
+  tasks:
+    - include_role:
+        name: storage
+
+    - include_tasks: get_unused_disk.yml
+
+    - name: Create a LVM logical volume with xfs fs_type
+      include_role:
+        name: storage
+      vars:
+        storage_pools:
+          - name: foo
+            disks: "{{ unused_disks[0] }}"
+            volumes:
+              - name: test1
+                size: 5g
+                mount_point: "{{ mount_location_before }}"
+
+    - include_tasks: verify-role-results.yml
+
+    - name: Change the file system signature on /dev/mapper/foo-test1
+      include_role:
+        name: storage
+      vars:
+        storage_pools:
+          - storage_pools:
```

Comment:
this needs to be removed

Reply:
what part exactly?

Reply:
just the redundant `- storage_pools:` under the first `storage_pools:`